### PR TITLE
Add test for Interpolate-1 layer

### PR DESCRIPTION
**Pull request by ElenaGvozdeva**
### Details:
 - There is test only for Interpolate-4 layer. Needs to add test for Interpolate-1 to test it

### Tickets:
 - *https://jira.devtools.intel.com/browse/EISW-11113*

### CODE REVIEW

**Review by iefode — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@ElenaGvozdeva do we really need new tests for release branch? I suppose they can be sent to master

**Comment by ElenaGvozdeva** [comment]
@ilya-lavrenov, I need this test for my Interpolate layer for kmb-plagin. Yesterday the development branch of kmb-plugin had been changed to releases/2021/4. So it needs to be add in releases/2021/4 OpenVino branch

File: inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/interpolate.hpp [hunk 1]
```diff
@@ -51,4 +51,27 @@ class InterpolateLayerTest : public testing::WithParamInterface<InterpolateLayer
     void SetUp() override;
 };
 
+//Interpolate-1 test
+typedef std::tuple<
+        InferenceEngine::Precision,        // Net precision
+        InferenceEngine::Precision,        // Precision
```
[the remark this answers is not in the document]
Comment (ElenaGvozdeva) [reply inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/interpolate.hpp:57]: It's the same for input and output. I just didn't want to add extra parameters

[the remark this answers is not in the document]
Comment (alexeytarakanov) [reply inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/interpolate.hpp:57]: May you could add note about it in comment? Just few words.

**Review by alexeytarakanov — APPROVED** [review]

**Comment by ElenaGvozdeva** [comment]
@iefode, could you merge this code?

**Pull request merged**